kabob-build/src/main/resources/edu/ucdenver/ccp/kabob/build/extensions/interaction-extension-classes.owl

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
```

Comment:
Thanks for the questions. I've updated the file after thinking about it some and consulting Mike. We are defining a new term, but now in a namespace separate from obo. It's now defined within http://ccp.ucdenver.edu/obo/ext. This is meant to be an extension of the OBOs and will allow queries such as the one you suggest above.

And yes, this file should get loaded during the build. I'll create a card for this.